app/helpers/middleware_server_group_helper/textual_summary.rb

```diff
@@ -0,0 +1,31 @@
+module MiddlewareServerGroupHelper::TextualSummary
+  include TextualMixins::TextualName
+  #
+  # Groups
```

Comment:
potentially a redundant comment - I am wondering what does it contribute to the code?